### PR TITLE
add telemetry environment tests

### LINKED ISSUE #180: Add Telemetry Environment tests
## Description
Add tests for hostName and basePath methods

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
closes #180

### CODE REVIEW

File: MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.TestFactory
+
+class TelemetryEnvironmentTest {
+
+    @TestFactory
+    fun `Check get host`() = TestCase {
+        Given("Object TelemetryEnvironment") {
+            mapOf(
+                China to "events.mapbox.cn",
```
Comment (korshaknn) [inline MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt]: lets's use constants

File: MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                }
+            }
+        }
+    }
+
+    @TestFactory
+    fun `Check base path`() = TestCase {
+        Given("Object TelemetryEnvironment") {
+            mapOf(
+                China to "Recordings_China",
```
Comment (korshaknn) [inline MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt]: constants here too

File: MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt (hunk 1 above)
Comment (alexander-kulikovskii) [reply MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt]: done

File: MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt (hunk 2 above)
Comment (alexander-kulikovskii) [reply MapboxVision/src/test/java/com/mapbox/vision/sync/FeatureEnvironment.kt]: done

**Pull request merged**